DigitalOffice/Tests/FileServiceTests/UnitTestLibrary/SerializerAssert.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FileServiceUnitTests.UnitTestLibrary
+{
+    public class SerializerAssert
+    {
+        
+    }
```

Comment:
I can not understand why we need this

Reply:
Hmm, I deleted this file and it should not be here

Reply:
File still here.

Reply:
@djino97 @DarkFelas I'm on their side on this one.
I can move a whole lot of code to the static class that will just take two arguments rather than write 5 or how-many-properties rows of code in whichever tests will be added to the FileService, for example.
My only problem with SerializerAssert was that I could not compare two Json objects because of the AddedOn property: each time it was different and never the same in both objects. 

Serializing objects to Json and then comparing is cool -- a few lines of code and we get a quick comparison.